clients/lsp-fsharp.el

```diff
@@ -179,7 +179,9 @@ available, else the globally installed tool."
 
 (defun lsp-fsharp--fsac-cmd ()
   "The location of fsautocomplete executable."
-  (or (expand-file-name "fsautocomplete" lsp-fsharp-server-install-dir)
+  (or (-let [maybe-local-executable (expand-file-name "fsautocomplete" lsp-fsharp-server-install-dir)]
```

Comment:
is there any particular need to have `-let` here instead of the regular `let` form?